style.css

```diff
@@ -623,3 +622,61 @@ button:active {
     align-items: center;
   }
 }
+
+.overlay-bg {
+  /* Height & width depends on how you want to reveal the overlay (see JS below) */
+  height: 100%;
+  width: 0;
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  background-color: #505ccacb;/* Black fallback color */
+  overflow-x: hidden; /* Disable horizontal scroll */
+  transition: 0.5s; /* 0.5 second transition effect to slide in or slide down the overlay (height or width, depending on reveal) */
+}
+
+.overlay-bg .closebtn {
+  padding: 0;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 0.299);
+  float: right;
+  margin: 24px;
+  transition: 0.3s;
+}
+
+/* Position the content inside the overlay */
+.overlay-content {
+  margin-top: 90px;
+}
+
+.overlay-content a {
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  display: block;
+  text-decoration: none;
+  transition: 0.3s;
+  padding: 8px;
+  margin-left: 20px;
+}
+
+/* When you mouse over the navigation links, change their color */
+.overlay-bg a:hover,
+.overlay-bg a:focus {
+  color: #f1f1f1;
+}
+
+/* When the height of the screen is less than 450 pixels, change the font-size of the links and position the close button again, so they don't overlap */
+@media screen and (max-height: 450px) {
+  .overlay-bg a {
+    font-size: 20px;
+  }
+
+  .overlay-bg .closebtn {
+    font-size: 40px;
+    top: 15px;
+    right: 35px;
+  }
+}
```

Comment:
- [ ] [optional] The mobile menu is too transparent, I would suggest you reduce the transparency so that it won't interfere the user experience
![Screenshot from 2021-12-14 09-46-47](https://user-images.githubusercontent.com/56886064/145954823-a3781be3-465a-4edc-bb36-0a7cceb272ee.png)